docs/auth/autologout.md

```diff
@@ -0,0 +1,96 @@
+      <Root>{routes}</Root>
+    </AutoLogoutProvider>
+    // ...
+  </>,
+);
+```
+
+## Configuration
+
+You can further adjust the Auto Logout settings by tweaking the available `<AutoLogoutProvider>` properties:
+
+```ts
+<AutoLogoutProvider
+  idleTimeoutMinutes={30}
+  useWorkerTimers={false}
+  logoutIfDisconnected={false}
+>
+  <Root>{routes}</Root>
+</AutoLogoutProvider>
+```
+
+If you prefer to have different settings for each Backstage instance deployed at your infrastructure, you can instead leverage the `<ConfigBasedAutoLogoutProvider>` which reads the Auto Logout settings from `app-config`.
+
+To do so, adjust your `App.tsx` as follows:
+
+```ts
+import { ConfigBasedAutoLogoutProvider } from '@backstage/core-components';
+
+// ... App.tsx contents
+
+export default app.createRoot(
+  <>
+    // ...
+    <ConfigBasedAutoLogoutProvider>
+      <Root>{routes}</Root>
+    </ConfigBasedAutoLogoutProvider>
+    // ...
+  </>,
+);
+```
+
+And add these lines into your `app-config`:
+
+```yaml
+auth:
+  autologout:
+    enabled: true
+```
+
+You will now be able to configure the AutoLogoutProvider through your `app-config`. These are the available settings:
+
+| Configuration Key                         | Component Property        | Description                                                                                                                                                                                                                                                            | Allowed Values                                                                                | Default Value                                                                                                                                                  |
+| ----------------------------------------- | ------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `auth.autologout.enabled`                 | `enabled`                 | Enable/disable the Auto Logout feature.                                                                                                                                                                                                                                | `true`/`false`                                                                                | `false`                                                                                                                                                        |
```

Comment:
code indicates that this is enabled by default so might wan't to change the Default value to true in this table.

Reply:
I realized docs are not so clear on this particular aspect.
The `<AutoLogoutProvider>` is, in fact, enabled by default. But if instead you are leveraging the `<ConfigBasedAutoLogoutProvider>` then if `auth.autologout.enabled` is not specified then the Auto logout will be disabled by default.
I'll clarify this, thanks :)